### ISSUE TITLE
JRE Load Error when attempting to start from macOS Executable

### ISSUE THREAD

**Issue by LampCo** [kevinmussi/iTunes-Discord-RP#9]
Launching it just gives a box that says "JRELoadError". The .jar version works fine.

**Comment by harpalassi** [comment]
Having the same issue on 10.14.6. I'll try the .jar though, thanks!

**Comment by kevinmussi** [comment]
I have never experienced this issue, so I have no clue why it would happen. Can you try running the app from the Terminal, and tell me what kind of error is shown on the console? (To run it just paste `/Applications/iTunesDiscordRP.app/Contents/MacOS/JavaAppLauncher` in the Terminal and hit Enter, assuming you have the app in the Applications folder).

**Comment by bermudalocket** [comment]
Same issue. Unfortunately there's no output when running from terminal before the alert appears. The application then quits quietly.

I'm on macOS 10.15.3 and use jenv. I tried running the app with literally every version of Java I have on my system right now:

```❯ jenv versions
  system
  1.8
  1.8.0.232
  12.0
  12.0.2
  13.0
  13.0.1
  openjdk64-1.8.0.232
  openjdk64-12.0.2
* openjdk64-13.0.1 (set by /Users/bermudalocket/.jenv/version)

**Comment by jalovisko** [comment]
I have the same issue. Running **macOS 10.15.3** with **Java 1.8.0_151**.

**Comment by iamrecursion** [comment]
Same issue on my end. MacOS 10.15.4 with various JDKs set via Jenv.

**Comment by kevinmussi** [comment]
Can all of you confirm that the standalone .jar app works fine, while the bundled .app shows this error?

**Comment by iamrecursion** [comment]
The standalone jar fails to start for me. Permissions error due to gatekeeper.

**Comment by bermudalocket** [comment]
I can only start the standalone jar by bypassing Gatekeeper (right-click -> open). However, [it fails to find Discord](https://i.imgur.com/XLonYot.png). If I build the project myself, it works.

**Comment by csd-oss** [comment]
> I can only start the standalone jar by bypassing Gatekeeper (right-click -> open). However, [it fails to find Discord](https://i.imgur.com/XLonYot.png). If I build the project myself, it works.

Same, but I haven't tried to build it myself.

**Comment by checcoux** [comment]
Macos 10.15.4 and the .app doesn't work

**Comment by TheMrIron2** [comment]
Bug still isn't fixed as of the latest version, 1.5.3. Experiencing the same symptoms as others here on the latest Java. JAR works.

**Comment by LampCo** [comment]
> Update: Fixed by reinstalling the latest JRE runtime and restarting. Everyone who is experiencing this should try to make sure?

This worked for me, it seems that it works fine with newer versions of JRE. Anyone still experiencing this issue should reinstall the latest JRE (8u251 is what worked for me)
